src/proxy/metrics.rs

```diff
@@ -27,6 +27,7 @@ use crate::state::workload::Workload;
 pub struct Metrics {
     pub connection_opens: Family<CommonTrafficLabels, Counter>,
     pub connection_close: Family<CommonTrafficLabels, Counter>,
+    pub connection_deny: Family<CommonTrafficLabels, Counter>,
```

Comment:
will take a closer look in a bit (and the other PR), but high level comment -- we have the "Core Istio metrics" and new metrics. The former is bound by backwards compat.

I wonder if for the new ones we should:
1. mark all of them experimental. Maybe in the metric description we just put `[EXPERIMENTAL]`, I think that is how k8s does it.
2. Use a reduced set of labels. We have like 5 labels that are ~the same, mostly for backwards compat. We don't really need all of them IMO.

WDYT?

Reply:
I'd definitely be open to that, do you have a list that you'd like to get rid of?